spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/src/test/resources/application-test.properties

```diff
@@ -0,0 +1,2 @@
+spring.cloud.gcp.spanner.instance-id=spring-demo
```

Comment:
Ditto.

Reply:
The other samples actually use their own test-properties files like this. I think it should've been this way originally. 

These are test files btw.